schema/content-descriptor.json

```diff
@@ -14,6 +14,13 @@
     },
     "digest": {
       "$ref": "defs-image.json#definitions/digest"
+    },
+    "urls": {
+      "description": "a list of urls from which this object may be downloaded",
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
These should use the [`uri` format][1]:

```json
"type": "string",
"format": "uri"
```

[1]: https://tools.ietf.org/html/draft-wright-json-schema-validation-00#section-7.3.6